### PR TITLE
Chinese Document

### PR DESCRIPTION
**Pull request by xjchengo**

### CODE REVIEW

**Comment by kazupon** [comment]
Oh!! Great!!  👍

**Comment by kazupon** [comment]
recently, in v2.1, I added the new feature and documentation.
https://github.com/vuejs/vue-validator/blob/dev/docs/en/classes.md

**Comment by xjchengo** [comment]
Okay, I will translate it today and refine other trans this weekend.

File: docs/zh-cn/structure.md [hunk 1]
```diff
@@ -61,24 +61,24 @@ Validation results can be accessed in this structure:
...
+- `untouched`: `touched` 的逆.
+- `modified`: 字段值与**初始**值不同时返回 `true`,否则返回 `false`。
+- `dirty`: 字段值改变过至少**一次**时返回 `true`,否则返回 `false`。
+- `pristine`: `dirty` 的逆.
+- `errors`: 字段无效时返回存有错误信息的数据，否则返回 `undefined`。
 
-## Top level validation properties
-- `valid`: whether **all** fields is valid. if so, then return `true`, else return `false`.
-- `invalid`: if invalid field exist even **one** in validate fields, return `true`, else `false`.
-- `touched`: whether **all** fields is touched, if so, return `true`, else `false`.
```
Comment (xjchengo) [inline docs/zh-cn/structure.md]: @kazupon Can you check if the explain for top level `touched` and `untouched` is correct? According to the source code, top level `touched` is true if there is any field that `touch` is true.

Comment (kazupon) [inline docs/zh-cn/structure.md]: @xjchengo 
Thank you for your feedback!!
I fixed english docs.
https://github.com/vuejs/vue-validator/commit/b48759ed2e38309983d63205d17ce9b98d76953b

**Comment by xjchengo** [comment]
@kazupon I have translated the doc of new feature and refined other docs. And i modified build config in https://github.com/vuejs/vue-validator/pull/196/commits/482167204eb7efe3baf5997a5d12117a66db4a7f to support multi language docs. Do I need to squash these commits into one or two commits before merging into dev?

**Comment by kazupon** [comment]
Thanks 💯

**Pull request merged**